Source/WebKit/UIProcess/API/Cocoa/_WKInspectorWindow.mm

```diff
@@ -24,16 +24,14 @@
  */
 
 #import "config.h"
-#import "_WKInspectorWindow.h"
+#import "_WKInspectorWindowInternal.h"
 
 #if PLATFORM(MAC)
 
 @implementation _WKInspectorWindow
 
-- (BOOL)isForRemoteTarget
-{
-    return _forRemoteTarget;
-}
+@synthesize forRemoteTarget = _forRemoteTarget;
+@synthesize inspectedWebView = _inspectedWebView;
```

Comment:
Are these properties not being automatically synthesized?

Reply:
I'm not sure. I'll remove the manual synthesis if that compiles. I was probably thinking (wrongly?) that auto synthesis isn't available when you're not using ARC.